packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -2786,63 +2802,6 @@ describe("typespec-client-generator-core: package", () => {
       strictEqual(bodyParameter.type.properties[0].name, correspondingMethodParams[0].name);
     });
 
-    it("spread with discriminate type with implicit property", async () => {
```

Comment:
this test has been removed because if we treat spread as spread, then spread a discriminated type is invalid.